widget/ui/src/components/Skeleton/Skeleton.stories.tsx

```diff
@@ -1,28 +1,28 @@
-import { Meta } from '@storybook/react';
+import type { PropTypes } from './Skeleton.types';
+import type { Meta } from '@storybook/react';
+
 import React from 'react';
-import { Skeleton, PropTypes } from './Skeleton';
+
+import { Divider } from '../Divider';
+
+import { Skeleton } from './Skeleton';
 
 export default {
   title: 'Components/Skeleton',
   component: Skeleton,
   args: {
-    width: 20,
-    height: 20,
-  },
-  argTypes: {
-    width: {
-      name: 'width',
-      control: { type: 'select' },
-      options: [20, 24, 36, 48],
-      defaultValue: 20,
-    },
-    height: {
-      name: 'height',
-      control: { type: 'select' },
-      options: [20, 24, 36, 48],
-      defaultValue: 20,
-    },
+    width: 50,
   },
 } as Meta<typeof Skeleton>;
 
-export const Main = (props: PropTypes) => <Skeleton {...props} />;
+export const Main = (props: PropTypes) => (
```

Comment:
It seems changing the prop values in Storybook doesn't work. Could you check it again?